packages/aws-cdk-lib/aws-ecs/lib/cluster.ts

```diff
@@ -69,9 +69,17 @@ export interface ClusterProps {
    * If true CloudWatch Container Insights will be enabled for the cluster
    *
    * @default - Container Insights will be disabled for this cluster.
+   * @deprecated See `containerInsightsLevel`
    */
   readonly containerInsights?: boolean;
 
+  /**
+   * The CloudWatch Container Insights configuration for the cluster
+   *
+   * @default - Container Insights configuration will not be set for the cluster
```

Comment:
A bit ambiguous, that does not indicate the actual default value. I'm assuming the insights would be disabled?

Reply:
The default is not entirely straightforward. If no setting is specified, no setting for container insights is set on the cluster, and the container insights setting is inherited from any account level ecs settings. Most likely,  container insights would be disabled, but that isn't guaranteed. I updated the default message to " Container Insights will be disabled for this cluster, but this may be overridden by ECS account level settings." If we'd prefer to just just say it's disabled, I'm fine with that as well.